libraries/botbuilder-lg/src/templatesParser.ts

```diff
@@ -478,21 +477,24 @@ export class TemplatesTransformer extends AbstractParseTreeVisitor<any> implemen
      * @private
      */
     private checkTemplateBody(
-        templateName: string,
-        templateBody: string,
+        template: Template,
         context: lp.TemplateBodyContext,
         startLine: number
     ): BodyContext {
-        if (templateBody === undefined || templateBody.trim() === '') {
+        if (template.body === undefined || template.body.trim() === '') {
             const diagnostic = this.buildTemplatesDiagnostic(
-                TemplateErrors.noTemplateBody(templateName),
+                TemplateErrors.noTemplateBody(template.name),
                 context,
                 DiagnosticSeverity.Warning
             );
             this.templates.diagnostics.push(diagnostic);
         } else {
             try {
-                return this.antlrParseTemplate(templateBody, startLine);
+                const templateBodyContext = this.antlrParseTemplate(template.body, startLine);
+                if (templateBodyContext) {
+                    template.templateBodyParseTree = templateBodyContext;
+                    new TemplateBodyTransformer(template).transform();
```

Comment:
Does the constructed instance need to be captured at all? It seems odd to invoke `new` without assigning it to something.